login.cpp

```diff
@@ -190,10 +189,10 @@ class Client {
 
 int main() {
   uestc::Config config{};
-  uestc::Client client("http://aaa.uestc.edu.cn");
-
-  uestc::UserInfo user(config.get("username"), config.get("password"), "", "1",
-                       uestc::kEncVer);
+  uestc::kDomain=config.get("kDomain");
+  uestc::Client client("http://10.253.0.237");
```

Comment:
The `IP` address is not the same with the as dorm telecom (10.253.0.235), so we need to replace `IP` to `URL`， and it will automatically resolve to the corresponding `IP`.